datafusion/datasource-parquet/src/source.rs

```diff
@@ -559,25 +549,8 @@ impl FileSource for ParquetSource {
                     .predicate()
                     .map(|p| format!(", predicate={p}"))
                     .unwrap_or_default();
-                let pruning_predicate_string = self
-                    .pruning_predicate
-                    .as_ref()
-                    .map(|pre| {
-                        let mut guarantees = pre
-                            .literal_guarantees()
-                            .iter()
-                            .map(|item| format!("{}", item))
-                            .collect_vec();
-                        guarantees.sort();
-                        format!(
-                            ", pruning_predicate={}, required_guarantees=[{}]",
-                            pre.predicate_expr(),
-                            guarantees.join(", ")
-                        )
-                    })
-                    .unwrap_or_default();
```

Comment:
In https://github.com/apache/datafusion/pull/15561#pullrequestreview-2741008179 Andrew asked me to keep this, but now since the schema isn't even being passed in to `with_predicate` it's going to be hard to keep these. I suggest we just accept that they won't be present in the physical plans. If that's not okay what I could do is generate them on the fly in `fmt_extra` or generate them if `with_predicate` is called with a schema or `with_schema` is called with a predicate. But I'd like to avoid that unless someone thinks is worth it or has another suggestion.

Reply:
I think it is important to keep these in the physical plans -- in particular what I think is important is to be able to check via the explain plan if pruning is happening by looking at the explain plan

Reply:
Hmm okay. I'll see if I can make it happen...